test/Sniff/fixtures/comments.less

```diff
@@ -0,0 +1,12 @@
+// Non-empty
+// 0
+//
+/*
+ * I'm not empty
+ */
+/*
+
+ */
+/*
+ *
+ */
```

Comment:
What about a single line comment containing a multi line or visa versa?
```
// /* asdffsd
```
or
```
/**
  // asdf
 */
```

Reply:
there are currently no style-rules that say that those two cases are not allowed. This is just checking if they are not empty.